Dockerfile.foreman

```diff
@@ -0,0 +1,18 @@
+FROM python:3.5-slim
+
+RUN groupadd user && useradd --create-home --home-dir /home/user -g user user
```

Comment:
Hmm perhaps, but I don't really know how much it matters because there will only ever be one user per container so we don't have to worry about them clashing with anything.